src/com/lern/homeWork/com/lern/homeWork/lesson6/extenseFigure/Rectangle.java

```diff
@@ -0,0 +1,48 @@
+package com.lern.homeWork.com.lern.homeWork.lesson6.extenseFigure;
+
+public class Rectangle extends Coordinates implements Area{
+
+    public Rectangle(Point leftTopPoint, Point rightBottomPoint) {
+        super(leftTopPoint, rightBottomPoint);
+    }
+
+    @Override
+    public double area() {
+        return Math.abs(getLeftTopPoint().getX() - getRightBottomPoint().getX()) *
+                Math.abs(getLeftTopPoint().getY() - getRightBottomPoint().getY());
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
+            return false;
+        }
+        final Rectangle other = (Rectangle) obj;
+        if (this.area() != other.area()) {
```

Comment:
if we have to Rectangle that Rectangle1.area = Rectangle2area but Rectangle1.length != Rectangle2.Width. Do you think that figores is equals?